wtformsparsleyjs/core.py

```diff
@@ -115,38 +131,40 @@ def _regexp_kwargs(kwargs, vali):
         regex_string = vali.regex.pattern
     else:
         regex_string = vali.regex
-    kwargs[u'data-parsley-regexp'] = regex_string
+    kwargs[u'data-parsley-pattern'] = regex_string
 
 def _url_kwargs(kwargs):
     kwargs[u'data-parsley-type'] = u'url'
 
-def _string_seq_delimiter(vali, kwargs):
-    # We normally use a comma as the delimiter - looks clean and it's parsley's default.
-    # If the strings for which we check contain a comma, we cannot use it as a delimiter.
-    default_delimiter = u','
-    fallback_delimiter = u';;;'
-    delimiter = default_delimiter
-    for value in vali.values:
-        if value.find(',') != -1:
-            delimiter = fallback_delimiter
-            break
-    if delimiter != default_delimiter:
-        kwargs[u'data-parsley-inlist-delimiter'] = delimiter
-    return delimiter
+def _anyof_kwargs(kwargs, vali):
+    # The inlist validator is no longer available in Parsley 2.x, so a custom anyof validator is used.
+    kwargs[u'data-parsley-anyof'] = u"[" + u",".join(vali.values) + u"]"
```

Comment:
I wonder if just using `json.dumps(vali.values)` here (and below) might be a little cleaner